modules/juce_audio_devices/native/juce_mac_CoreAudio.cpp

```diff
@@ -789,6 +789,22 @@ class CoreAudioInternal  : private AsyncUpdater
         const auto numInputChans  = getChannels (inStream);
         const auto numOutputChans = getChannels (outStream);
 
+        // With Bluetooth headset microphone, the buffer from macOS may be limited smaller than configured buffer size.
+        // For non-Bluetooth inputs and outputs changing sample rate does not change buffer size, and buffer size
+        // doesn't seem to be increased. The data here format is always 32-bit float so byte size is divided by 4 to get
+        // frame count even if bitDepth is something else than 32.
+        int thisBufferSize = 0;

```

Comment:
my only concern here is the naming of the variable, maybe something like `currentBufferSize`, or `calculatedBufferSize` instead of `thisBufferSize`?